doc/help/CP.md

```diff
@@ -0,0 +1,7 @@
+kdb-cp(1) -- Copy keys within the key database
+================================
+
+## USAGE
+`kdb cp <source> <dest>`  
```

Comment:
Also for this trivial things we can give more explanation (e.g. what happens if key already exists) and some examples.